packages/tools/kadena-cli/src/prompts/tx.ts

```diff
@@ -508,6 +512,19 @@ export async function selectSignMethodPrompt(): Promise<'wallet' | 'keyPair'> {
   });
 }
 
+function determineNetwork(networkId: string | null): string {
+  const id = networkId ?? '';
+
+  if (id.includes('testnet')) {
+    return 'testnet';
+  } else if (id.includes('mainnet')) {
+    return 'mainnet';
+  } else if (id.includes('devnet')) {
+    return 'devnet';
```

Comment:
network ids are different than the alias/network names, so relying on the ids like testnet04, mainnet01 and development would be better. isn't it?